helios-tools/pom.xml

```diff
@@ -58,6 +58,10 @@
       <groupId>com.typesafe</groupId>
       <artifactId>config</artifactId>
     </dependency>
+    <dependency>
+      <groupId>javax.ws.rs</groupId>
+      <artifactId>jsr311-api</artifactId>
```

Comment:
I could be wrong, but if this dependency is for things like the `@GET` annotation then I think the better dependency is `javax.xml.ws:jaxws-api`